ext/fetch/23_response.js

```diff
@@ -181,7 +181,9 @@ function initializeAResponse(response, init, bodyWithType) {
     init.statusText &&
     RegExpPrototypeExec(REASON_PHRASE_RE, init.statusText) === null
   ) {
-    throw new TypeError("Status text is not valid.");
+    throw new TypeError(
+      `Status text is not valid: received "${init.statusText}"`,
```

Comment:
I kinda like the more direct phrasing of the Header errors like: `Invalid header value: ...` 

rather than saying `XY is not valid: <reason>`.

```suggestion
      `Invalid status text: "${init.statusText}"`,
```

Reply:
I like this as it's concise. There is a rule in the style guide that says
> Message should state the action that lead to the error

Although it's sometimes not clear and repeating the function such as "Failed to Initialize A Response as status text is not valid" is way to terse. +1 for these suggestions.